### PR TITLE
add eau75

### PR DESCRIPTION
**Pull request by jiaxin96**
## Description
Add eau_r2 keyboard.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/yandrstudio/eau_r2/rules.mk [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes           # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+RGBLIGHT_DRIVER = WS2812    # RGB driver support
```
Comment (fauxpark) [inline keyboards/yandrstudio/eau_r2/rules.mk:15]:
```suggestion
```

File: keyboards/yandrstudio/eau_r2/eau_r2.c [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "eau_r2.h"
+
+void keyboard_post_init_kb(void) {
+#   ifdef RGBLIGHT_ENABLE
+    rgblight_reload_from_eeprom();
+#   endif
+}
+
```
Comment (fauxpark) [inline keyboards/yandrstudio/eau_r2/eau_r2.c:23]:
```suggestion
```

File: keyboards/yandrstudio/eau_r2/info.json [hunk 3]
```diff
@@ -0,0 +1,444 @@
+{
+    "manufacturer": "CHAOSERA_Y&R",
+    "keyboard_name": "eau75",
+    "processor": "STM32F103",
+    "bootloader": "stm32duino",
+    "usb": {
+        "pid": "0xAACD",
```
Comment (fauxpark) [inline keyboards/yandrstudio/eau_r2/info.json:7]: Missing VID.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/yandrstudio/eau_r2/info.json (hunk 3 above)
Comment (jiaxin96) [reply keyboards/yandrstudio/eau_r2/info.json:7]: VID is in info.json of the root studio directory, "https://github.com/qmk/qmk_firmware/blob/master/keyboards/yandrstudio/info.json#L6"

File: keyboards/yandrstudio/eau_r2/eau_r2.c (hunk 2 above)
Comment (jiaxin96) [reply keyboards/yandrstudio/eau_r2/eau_r2.c]: In some STM32 cases, adding this code can avoid the abnormal lighting of the first RGB, and using PWM as the driver can also avoid this problem.

File: keyboards/yandrstudio/eau_r2/info.json [hunk 4]
```diff
@@ -0,0 +1,444 @@
+{
+    "manufacturer": "CHAOSERA_Y&R",
```
Comment (fauxpark) [inline keyboards/yandrstudio/eau_r2/info.json:2]:
```suggestion
```
This won't be applied due to the order in which info.json files are combined.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**